### ISSUE TITLE
[request https://www.samehadaku.net/ ]

### ISSUE THREAD

**Issue by evetaku** [reek/anti-adblock-killer#3477]
https://www.samehadaku.net/
![samehada](https://user-images.githubusercontent.com/6820983/28952194-d77462f0-78f9-11e7-9dd6-0aac5b8ee363.png)

mozilla 50.0.1
gresasemonkey 3.11
ublock origin 1.13.8 and adblocker ultimate 

Thanks

**Comment by uBlock-user** [comment]
Anti adblock script

```javascript
if( tie.ad_blocker_detector && typeof $tieE3 == 'undefined' ){
		adBlock = true;
		$html.css({'marginRight': getScrollBarWidth(), 'overflow': 'hidden'});
		setTimeout(function(){ $body.addClass('tie-popup-is-opend')},10);
		tieBtnOpen('#tie-popup-adblock');
	}
```

https://www.samehadaku.net/wp-content/themes/jannah/js/scripts.js

**Comment by uBlock-user** [comment]
```javascript
if ( $tiePopup.length && ! adBlock ){
    $doc.keyup(function(event){
			if ( event.which == '27' && $body.hasClass('tie-popup-is-opend')){
        tie_close_popup();
      }
    });
```

**Comment by uBlock-user** [comment]
Filter - `samehadaku.net##script:inject(abort-on-property-read.js, tie.ad_blocker_detector)`

@jspenguin2017

**Comment by jspenguin2017** [comment]
This one is caught in generic anti-bait filter.

**Comment by jspenguin2017** [comment]
But OK, I'll have a script rule so we aren't that relied on generic anti-bait.

**Comment by uBlock-user** [comment]
They changed something as the generic solution doesn't work on this. I can reproduce it.

**Comment by uBlock-user** [comment]
Were you able to reproduce it ?

**Comment by jspenguin2017** [comment]
No. I can't reproduce it, the bait file is caught in `@@/advertisement*.js$first-party,script`.

**Comment by uBlock-user** [comment]
Ah, yes the whitelisted filter, I ` $badfilter`ed them all, that's why I can reproduce.

**Comment by kurieb** [comment]
if sricpt for Chrome, Tampermonkey, ublock origin 1.13.8 to https://www.samehadaku.net/ ?

**Comment by Samehadakunet** [comment]
Dasar lucknut mau gratis mau instant... pas hampir semua fansub pada tutup nyesel lu....
